ktor-server/ktor-server-plugins/ktor-server-i18n/jvm/test/io/ktor/tests/i18n/TranslationTest.kt

```diff
@@ -107,4 +109,10 @@ class TranslationTest {
         val contentAsString = response.bodyAsText()
         assertEquals("Русский Ключ", contentAsString)
     }
+
+    /**
+     * On Java 8 PropertyResourceBundle requires properties to be encoded in ISO-8859-1,
+     * while starting from Java 9 the default encoding is UTF-8.
+     */
+    private fun String.fixJava8Encoding(): String = this.toByteArray(Charsets.ISO_8859_1).toString(Charsets.UTF_8)
```

Comment:
Should this "fix" be applied inside `i18n` implementation? Or we assume that users know that bundles should be in a different encoding on Java 8?